aspnetcore/host-and-deploy/linux-nginx.md

```diff
@@ -393,10 +483,24 @@ Add the */etc/nginx/proxy.conf* configuration file:
 
 [!code-nginx[](linux-nginx/proxy.conf)]
 
+# [Ubuntu](#tab/linux-ubuntu)
+
 **Replace** the contents of the */etc/nginx/nginx.conf* configuration file with the following file. The example contains both `http` and `server` sections in one configuration file.
 
 [!code-nginx[](linux-nginx/nginx.conf)]
```

Comment:
This one needs to be deleted since it's been moved to be outside the tabs. Otherwise, the Ubuntu tab is showing two of the same code blocks.

Reply:
Ouch!  Good catch!